src/libawkward/forth/ForthMachine.cpp

```diff
@@ -61,49 +61,50 @@ namespace awkward {
   #define CODE_LEN_OUTPUT 22
   #define CODE_REWIND 23
   // generic builtin instructions
-  #define CODE_PRINT_STRING 24
-  #define CODE_PRINT 25
-  #define CODE_PRINT_CR 26
-  #define CODE_PRINT_STACK 27
-  #define CODE_I 28
-  #define CODE_J 29
-  #define CODE_K 30
-  #define CODE_DUP 31
-  #define CODE_DROP 32
-  #define CODE_SWAP 33
-  #define CODE_OVER 34
-  #define CODE_ROT 35
-  #define CODE_NIP 36
-  #define CODE_TUCK 37
-  #define CODE_ADD 38
-  #define CODE_SUB 39
-  #define CODE_MUL 40
-  #define CODE_DIV 41
-  #define CODE_MOD 42
-  #define CODE_DIVMOD 43
-  #define CODE_NEGATE 44
-  #define CODE_ADD1 45
-  #define CODE_SUB1 46
-  #define CODE_ABS 47
-  #define CODE_MIN 48
-  #define CODE_MAX 49
-  #define CODE_EQ 50
-  #define CODE_NE 51
-  #define CODE_GT 52
-  #define CODE_GE 53
-  #define CODE_LT 54
-  #define CODE_LE 55
-  #define CODE_EQ0 56
-  #define CODE_INVERT 57
-  #define CODE_AND 58
-  #define CODE_OR 59
-  #define CODE_XOR 60
-  #define CODE_LSHIFT 61
-  #define CODE_RSHIFT 62
-  #define CODE_FALSE 63
-  #define CODE_TRUE 64
+  #define CODE_STRING_ERROR 24
```

Comment:
Maybe just `CODE_STRING` because all that

```forth
s" Hello world!"
```

does is put an index of the static `"Hello world!"` string onto the stack. You'll be using it for communicating error messages from Forth to TypedArrayBuilder, but it could have other uses as well.